redbeams-api/src/main/java/com/sequenceiq/redbeams/api/endpoint/v4/database/request/OracleParameters.java

```diff
@@ -0,0 +1,11 @@
+package com.sequenceiq.redbeams.api.endpoint.v4.database.request;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+
+import io.swagger.annotations.ApiModel;
+
+@ApiModel
+@JsonIgnoreProperties(ignoreUnknown = true)
+public class OracleParameters extends DatabaseRequestParameters {
```

Comment:
I commented out the Oracle stuff for database servers. I think we can avoid including it for now, because we are targeting only PostgreSQL for the near future.

Reply:
Deleted.